src/css-grids/cssgrids.css

```diff
@@ -0,0 +1,185 @@
+/* Reset & Utilities */
+*{
+   padding: 0;
+   margin: 0;
+   box-sizing: border-box;
+}
+body{
+   font-family: Arial, Helvetica, sans-serif;
+   color: #ccc;
+   background: rgb(20, 0, 0);
+   width: 100vw;
+   height: 100vh;
+   position: relative;
+   text-shadow: 3px 3px rgb(46, 38, 2);
+}
+body::before{
+   content: "";
```

Comment:
danggggg. i've never done it like this before! clever way to darken this image. I'm stealing. 
also look into the filter property. It's a lot of fun
- [https://css-tricks.com/almanac/properties/f/filter/](url)
- [https://developer.mozilla.org/en-US/docs/Web/CSS/filter](url)